client/components/domains/domain-product-price/index.jsx

```diff
@@ -87,7 +87,27 @@ class DomainProductPrice extends React.Component {
 		);
 	}
 
+	renderSalePrice() {
+		const { price, salePrice, translate } = this.props;
+
+		return (
+			<div className={ classnames( 'domain-product-price', 'is-free-domain' ) }>
+				<div className="domain-product-price__free-text">{ salePrice }</div>
+				<div className="domain-product-price__price">
+					{ translate( 'Renews at: %(cost)s {{small}}/year{{/small}}', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Renews in one year at: %(cost)s {{small}}/year{{/small}}' )` *ES Score: 16*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/display-domain-sale-flag)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).